docs/book/deployment.md

```diff
@@ -309,6 +309,51 @@ ignore any file types it does not recognize. These files must have the file exte
 `.png`, `.jpeg`, `.jpg`, `.mp3`, `.json`, or`.txt`. Litexa extensions may add to that list.
 Please see the [section on assets](/book/presentation.html#asset-file-references) for more information.
 
+:::tip Overriding a deployment's assets root path
+You can optionally point a deployment at an arbitrary location, and Litexa
+will blindly expect the assets to be there. This is useful if you have a massive
+project and don't want every developer to push their own isolated asset caches up,
+or even pull the assets onto their machine in the first place.
```

Comment:
Would slightly reword to something like:

You can override the skill's assets URL path, instead of using the default S3 path (location of files deployed from `litexa/assets`). This can be useful when collaborating on a project with sizable assets, to prevent each contributor needing to upload and maintain their own copies of assets.